internal/cmd/commands/workerscmd/funcs.go

```diff
@@ -25,12 +54,103 @@ func (c *Command) extraHelpFunc(helpMap map[string]func() string) string {
 			"",
 			"  Please see the workers subcommand help for detailed usage information.",
 		})
+	case "add-worker-tags":
+		helpStr = base.WrapForHelpText([]string{
+			"Usage: boundary workers add-worker-tags [sub command] [args]",
+			"",
+			"  This command allows adding api tags to worker resources. Example:",
+			"",
+			"    Add a set of api tags to a specified worker:",
+			"",
+			`      & boundary workers add-worker-tags -id w_1234567890 -tag "key1=val-a" -tag "key2=val-b,val-c"`,
+			"",
+			"",
+		})
+	case "set-worker-tags":
+		helpStr = base.WrapForHelpText([]string{
+			"Usage: boundary workers set-worker-tags [sub command] [args]",
+			"",
+			"  This command allows setting api tags for worker resources. Example:",
+			"",
+			"    Set api tags for a specified worker:",
+			"",
+			`      & boundary workers set-worker-tags -id w_1234567890 -tag "key1=val-a" -tag "key2=val-b,val-c"`,
+			"",
+			"",
+		})
+	case "remove-worker-tags":
+		helpStr = base.WrapForHelpText([]string{
+			"Usage: boundary workers remove-worker-tags [sub command] [args]",
+			"",
+			"  This command allows removing api tags from worker resources. Example:",
+			"",
+			"    Remove a set of api tags to a specified worker:",
+			"",
+			`      & boundary workers remove-worker-tags -id w_1234567890 -tag "key1=val-a" -tag "key2=val-b,val-c"`,
+			"",
+			"",
+		})
 	default:
 		helpStr = helpMap[c.Func]()
 	}
 	return helpStr + c.Flags().Help()
 }
 
+func extraFlagsFuncImpl(c *Command, _ *base.FlagSets, f *base.FlagSet) {
+	for _, name := range flagsMap[c.Func] {
+		switch name {
+		case "tag":
+			var nullCheckFn func() bool = nil
+			switch c.Func {
+			case "set-worker-tags":
+				nullCheckFn = func() bool { return true }
+			default:
+			}
+			f.StringSliceMapVar(&base.StringSliceMapVar{
+				Name:      "tag",
+				Target:    &c.FlagTags,
+				NullCheck: nullCheckFn,
+				Usage:     "The api tag resources to add, remove, or set.",
+			})
+		}
+	}
+}
+
+func extraFlagsHandlingFuncImpl(c *Command, _ *base.FlagSets, opts *[]workers.Option) bool {
+	switch c.Func {
+	case "add-worker-tags", "remove-worker-tags":
```

Comment:
This is missing a `set-worker-tags` case.

Reply:
Do we want an empty `-tag` entry to convert to nil for set-worker-tags? I thought it was pretty nifty when `boundary set-worker-tags -id w_idxxxxxxxx` cleared the tags

Reply:
I added a `set-worker-tags` case with `No tag supplied via -tag. Please use "-tag null" to clear all tags` as the UI error when there is no -tag input

Reply:
Actually yeah I'm not sure what to do here since if I check whether `c.FlagTags` is empty, it overrides the "null" case--where I set target to `nil` for a `stringSliceMapValue` in `flags.go`--and returns an error. 

I don't want to set a token entry for the `map[string][]string` to represent that its value should be nil, since that limits a valid key from use (even if it's a mapping like `null=true`). I don't like the idea of using an invalid key with commas like some form of `,,null,,` since I think that's a bit confusing and unintuitive.

The most clear way forward might be creating another/using another existing bool to distinguish when the user inputs a "null" entry for `set-worker-tags`..

However, do I have to put a `set-worker-tags` case here? Is it alright for the user to be able to null the entry via `boundary set-worker-tags -id w_idxxxxxxxx` and `boundary set-worker-tags -id w_idxxxxxxxx -tag null`?